src/components/App.js

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
 import Display from './display';
-import Button from './button';
 import ButtonPanel from './buttonPanel';
 
 export default class App extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = { calculation: 0 };
+    this.state = {
+      calculation: 0,
+    };
   }
+
   render() {
-    const calculation = this.state
-    return(
-      <section>
-        <div>
+    const { calculation } = this.state;
 
-        </div>
+    return (
+      <section className="app">
+        <Display result={calculation} />
+        <ButtonPanel />
       </section>
```

Comment:
**REQUIRED** 
* Has children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))